modules/sdk-core/src/bitgo/utils/tss/ecdsa/typesMPCv2.ts

```diff
@@ -0,0 +1,40 @@
+import * as t from 'io-ts';
+import {
+  MPCv2KeyGenRound1Request,
+  MPCv2KeyGenRound1Response,
+  MPCv2KeyGenRound2Request,
+  MPCv2KeyGenRound2Response,
+  MPCv2KeyGenRound3Request,
+  MPCv2KeyGenRound3Response,
+} from '@bitgo/public-types';
+
+export {
+  MPCv2KeyGenRound1Request,
+  MPCv2KeyGenRound1Response,
+  MPCv2KeyGenRound2Request,
+  MPCv2KeyGenRound2Response,
+  MPCv2KeyGenRound3Request,
+  MPCv2KeyGenRound3Response,
+} from '@bitgo/public-types';
```

Comment:
Why are we re-exporting this?

Reply:
instead of this, install public-types as dev depencies in bitgo for tests